.jenkins/pytorch/test.sh

```diff
@@ -459,6 +459,9 @@ test_bazel() {
 
   get_bazel
 
+  # Test //c10/... in the default mode.
+  tools/bazel test --test_timeout=480 --test_output=all --test_tag_filters=-gpu-required --test_filter=-*CUDA //c10/...
```

Comment:
Hmm, aren't `:all_tests` target already includes c10 tests?

Reply:
You are correct: it does already. This change makes more sense in the context of #69995 which runs the same tests with glog and gflags not enabled. This line makes that we run the same tests with glog and gflags more explicit.

Note that because of Bazel caching, we do not repeat the build or tests when we run it again under :all_tests. However, this *could* still be slightly less efficient because we lose some opportunities for parallelism when we are running strictly fewer targets. I'm not personally that concerned about that though.

Reply:
For the record, I am also ok with dropping this and clarifying it with a comment.